lib/core/versions/latest/CreateOperation.ts

```diff
@@ -121,67 +91,6 @@ export default class CreateOperation implements OperationModel {
 
     const didUniqueSuffix = Did.computeUniqueSuffix(suffixData);
 
-    const encodedSuffixData = Encoder.encode(JsonCanonicalizer.canonicalizeAsBuffer(suffixData));
-    return new CreateOperation(operationBuffer, didUniqueSuffix, encodedSuffixData, suffixData, encodedDelta, delta);
-  }
-
-  /**
-   * Parses the given operation object as a `CreateOperation`.
-   * The `operationBuffer` given is assumed to be valid and is assigned to the `operationBuffer` directly.
-   * NOTE: This method is purely intended to be used as an optimization method over the `parse` method in that
-   * JSON parsing is not required to be performed more than once when an operation buffer of an unknown operation type is given.
-   * @param coreIndexFileMode If set to true, then `delta` and `type` properties are expected to be absent.
-   */
-  public static async parseObject (operationObject: any, operationBuffer: Buffer, coreIndexFileMode: boolean): Promise<CreateOperation> {
-    // TODO: SIP 2 #781 deprecates this. Should be deleted when fully switched over
-    let expectedPropertyCount = 3;
-    if (coreIndexFileMode) {
-      expectedPropertyCount = 1;
-    }
-
-    const properties = Object.keys(operationObject);
-    if (properties.length !== expectedPropertyCount) {
-      throw new SidetreeError(ErrorCode.CreateOperationMissingOrUnknownProperty);
-    }
-
-    const encodedSuffixData = operationObject.suffixData;
-    const suffixData = await CreateOperation.parseSuffixData(encodedSuffixData);
-
-    // If not in core index file mode, we need to validate `type` and `delta` properties.
-    let encodedDelta;
-    let delta;
-    if (!coreIndexFileMode) {
-      if (operationObject.type !== OperationType.Create) {
-        throw new SidetreeError(ErrorCode.CreateOperationTypeIncorrect);
-      }
-
-      encodedDelta = operationObject.delta;
-      try {
-        delta = await Operation.parseDelta(operationObject.delta);
-      } catch {
-        // For compatibility with data pruning, we have to assume that `delta` may be unavailable,
-        // thus an operation with invalid `delta` needs to be processed as an operation with unavailable `delta`,
-        // so here we let `delta` be `undefined`.
-      }
-    }
-
-    const didUniqueSuffix = CreateOperation.computeDidUniqueSuffix(operationObject.suffixData);
-    return new CreateOperation(operationBuffer, didUniqueSuffix, encodedSuffixData, suffixData, encodedDelta, delta);
-  }
-
-  private static async parseSuffixData (suffixDataEncodedString: any): Promise<SuffixDataModel> {
-    if (typeof suffixDataEncodedString !== 'string') {
-      throw new SidetreeError(ErrorCode.CreateOperationSuffixDataMissingOrNotString);
```

Comment:
Error code no longer used anywhere, remove from `ErrorCode`, do this for all occurrences in this PR.
I thought about automating this in the error code generation script 'coz I am sure there are many, but haven't got around to it.